### ISSUE TITLE
How to use the `--auto` option in PRS-CSx?

### ISSUE THREAD

**Issue by goodluckncb** [getian107/PRScsx#50]
Hi,

Thank you for developing PRS-CSx, it's been incredibly helpful for my work. I have a question about the `--auto` option mentioned in the documentation.

1. Could you provide more details on how to correctly use the `--auto` option in PRS-CSx? Specifically, what arguments are required, and what steps should I follow to apply the algorithm with this option?

2. Additionally, what is the key difference between using the `--auto` option and the `--meta=True` option in terms of the algorithm and the assumptions about the populations? I understand that `--auto` does not require a validation dataset for tuning hyper-parameters, but could you explain the underlying principle behind this approach compared to the `meta` approach?

Thank you in advance for your time and assistance!

Best regards,  
caibo

**Comment by goodluckncb** [comment]
Thank you so much for your prompt and detailed response. I greatly appreciate your clarification on the 'auto' and 'meta' options in PRS-CSx.

I would like to confirm whether the following command is correct for running PRS-CSx with both 'auto' and 'meta' activated:

PRScsx.py \
    --ref_dir=/share/NCB/genotypes/PRS-CS/1KG \
    --bim_prefix=test \
    --sst_file=EUR_sumstats.txt,EAS_sumstats.txt \
    --n_gwas=200000,100000 \
    --pop=EUR,EAS \
    --chrom=22 \
    --meta=True \
    --out_dir=/share/NCB/CZL/code/PRScsx_code/PRScsx-master/test_data \
    --out_name=test
Could you kindly confirm if this command is correct, or if there are any adjustments I should make?

Additionally, I am conducting a multi-ancestry meta-analysis for a specific disease, using nearly all publicly available datasets. One of the reviewers raised a concern that my PRS analysis focused only on UK Biobank data, which is predominantly of European descent. At present, I only have genotype data from UKB. I am considering splitting the UKB cohort into a validation set and a test set, using a 9:1 split (with 9 for validation and 1 for testing). Do you think this split is appropriate, or would you recommend a different approach to prevent potential overfitting?

Moreover, I am wondering if it's feasible to utilize the non-European subset within UKB, which is relatively small, to predict PRS performance in other ancestry groups. Would this approach provide meaningful insight into cross-ancestry predictions, or are there potential limitations I should be aware of?

Thank you again for your time and valuable insights!

Best regards,
Caibo

**Comment by getian107** [comment]
Hi Caibo -- The code looks good to me. The non-European subset of UKBB may be used to evaluate the PRS in non-European populations if you have a reasonable number of cases in those datasets. Usually, 500 or more cases would provide reasonable power for PRS assessment.
